markus/backends/statsd.py

```diff
@@ -0,0 +1,97 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+import logging
+
+from statsd import StatsClient
+from markus.backends import BackendBase
+
+
+logger = logging.getLogger(__name__)
+
+
+class StatsdMetrics(BackendBase):
+    """Uses pystatsd client for statsd pings.
+
+    This requires the pystatsd module and requirements to be installed.
+    To install those bits, do::
+
+        $ pip install markus[statsd]
+
+
+    To use, add this to your backends list::
+
+        {
+            'class': 'markus.backends.statsd.StatsdMetrics',
+            'options': {
+                'statsd_host': 'statsd.example.com',
+                'statsd_port': 8125,
+                'statsd_prefix': None,
+                'statsd_maxudpsize': 512,
+            }
+        }
+
+
+    Options:
+
+    * statsd_host: the hostname for the statsd daemon to connect to
+
+      Defaults to ``'localhost'``.
+
+    * statsd_port: the port for the statsd daemon to connect to
+
+      Defaults to ``8125``.
+
+    * statsd_prefix: the prefix to use for statsd data
+
+      Defaults to ``None``.
+
+    * statsd_maxudpsize: the maximum data to send per packet
+
+      Defaults to ``512``.
+
+
+    .. seealso::
+
+       http://docs.datadoghq.com/guides/metrics/
```

Comment:
This seealso should probably be a different url so people who follow it don't get confused by Datadog statsd enhancements.

Maybe this url?: http://statsd.readthedocs.io/en/v3.2.1/configure.html